themes/default/content/learn/pulumi-fundamentals/configure-and-provision/index.md

```diff
@@ -92,8 +146,6 @@ mongo_image = docker.RemoteImage("mongo", name="mongo:bionic")
 Try and run your `pulumi up` again at this point. You should get an error like
 this:
```

Comment:
The error seems to have disappeared for the TS version. Does the TS version have a different error? If it doesn't, I think removing 149 and 159 should do it.

Reply:
Ah thanks, no, that's my oversight -- the same error occurs with TS. I'll fix, thanks for spotting.